app/js/event/event.sort.js

```diff
@@ -32,7 +32,7 @@
           getDistanceFromLatLonInKm(Number(events[i].latitude),
                                     Number(events[i].longitude),
                                     Number(coordinates.latitude),
-                                    Number(coordinates.longitude));
+                                    Number(coordinates.longitude)).toFixed(2);
```

Comment:
It's kinda confusing when you reply by editing my comments. You can add your own comments on this page https://github.com/APT-project/upcoming/pull/45/files

What you are suggesting is going to lead to cluttered, unmaintainable code in the long run. The factory should not format the data. The filter that you wrote is supposed to do that. The reason the filter is not working is because you have not included it in the `index.html` and are not using the filter.